oci/private/pull.bzl

```diff
@@ -494,13 +495,16 @@ def _oci_alias_impl(rctx):
         util.warning(rctx, """\
 for reproducible builds, a digest is recommended.
 Either set 'reproducible = False' to silence this warning,
-or run the following command to change oci_pull to use a digest:
+or run the following command to change {rule} to use a digest:
+{warning}
 
-buildozer 'set digest "sha256:{digest}"' 'remove tag' 'remove platforms' {optional_platforms} WORKSPACE:{name}
+buildozer 'set digest "sha256:{digest}"' 'remove tag' 'remove platforms' {optional_platforms} {location}
     """.format(
-            name = rctx.attr.name,
+            location = "MODULE.bazel:" + rctx.attr.name.split("~")[-1] if is_bzlmod_enabled() else "WORKSPACE:" + rctx.attr.name,
```

Comment:
Just note that a user may still have WORKSPACE repos of this type even if Bzlmod is enabled. 

Instead, you could add a new attribute that you only populated from the module extension as an indicator that the repo is coming from `MODULE.bazel`. You can also use this attribute to pass in the friendly repo name directly instead of reverse engineering it here.

Reply:
never mind, the repository name can already be found as a string argument in this location

Reply:
It looks like you could pass in this attribute unconditionally instead of only with Bzlmod, then you don't need to use the detection logic.